src/parser/evaql/evaql_parser.g4

```diff
@@ -253,6 +253,7 @@ selectElement
 
 fromClause
     : FROM tableSources
+      sample=sampleClause?
```

Comment:
Rethinking this design choice. Do you think the sampling should be done at the table level or query level? Right now we are doing it at the query level because we have a single sample value for the entire query. What if we want different sample values for different table sources? 
@xzdandy @karan-sarkar what are your thoughts on this design?

Reply:
Do you mean put the sample on the load statement?

Reply:
Could you give an example query in your mind?